src/datasets/non_desktop.md

```diff
@@ -0,0 +1,10 @@
+# Non-Desktop Data
+
+Non-Desktop data includes data for all mobile browser products as well as non browser products such as `Lockwise`, `echo`, `vpn`, `webxrviewer` among others.   
+
+## List of Datasets
+
+- [`Day 2-7 Activation`](./non_desktop/day_2_7_activation/reference.md) - Used to calculate the day 2-7 activation metric, a key result in 2020 for non-desktop products.
+- [`Google Play Store Data`](./non_desktop/google_play_store/reference.md) - Used to understand the acquisition performance for non-desktop products on the Google Play Store.
+- [`Apple App Store Data`](./non_desktop/apple_app_store/reference.md) - Used to understand the acquisition performance for the non-desktop products on the Apple App Store.
```

Comment:
We don't need to fence these items, as they're not actually table names. Also, let's link to the activation metric! That will help us connect the dots.

```suggestion
- [Day 2-7 Activation](./non_desktop/day_2_7_activation/reference.md) - Used to calculate the [Day 2-7 Activation metric](../metrics/metrics.md#day-2-7-activation), a key result in 2020 for non-desktop products.
- [Google Play Store Data](./non_desktop/google_play_store/reference.md) - Used to understand the acquisition performance for non-desktop products on the Google Play Store.
- [Apple App Store Data](./non_desktop/apple_app_store/reference.md) - Used to understand the acquisition performance for the non-desktop products on the Apple App Store.
```